contents/blog/dev-marketing-paid-ads.md

```diff
@@ -0,0 +1,154 @@
+- It is very easy to do ads that stand out here because 90% are so boring and corporate.
+- Good paid channel for reaching founder-type devs - and people who are more self-promotery, basically.
+- Expensive but hands-down the best targeting of any platform. 
+
+### Reddit
+
+- Very cheap but poor for awareness or conversion. Very rudimentary ad tools.
+- Instead, invest the time in being an active member of relevant subreddits where you contribute actual value to those people. This takes a lot more work.
+- Redditors hate ads nearly as much as HN readers do. Nearly. 
+
+### Quora
+
+- Dark horse - good for conversion and awareness.
+- Quite cheap, good targeting.
+- Seriously, I don't know why more people don't use Quora. 
+
+### Product Hunt
+
+- Good for awareness and conversion _if_ you have a cheap (<$10/mo) product.
+- Full of indiehackers/entrepreneurs/wannapreneurs, most of whom do not want to spend any money.
+- Kinda ok targeting but the majority of the audience is non-technical - tread carefully. 
+
+### Carbon Ads
+
+- Expensive. Lots of people rave about them. We don't.
+- We got neither awareness nor conversion when we ran campaigns there.
+
+### Bing
+
+- Fun fact - this is the only route to advertise on DuckDuckGo!
+- Unfun fact - you can only advertise on the Bing ads network, and they decide where the ads go. You can't say 'serve ads on DuckDuckGo only'.
+- Quite cheap. Good only if you want to target users at large enterprises where they are forced to use Bing. 
+
+### Sponsoring newsletters
+
+Not 'paid ads' in the traditional sense, but I wanted to share a few creators who write genuinely good content and we've had success working with. You're looking at $1,000-$10,000 per issue depending on reach. Aim to do short bursts of activity, not long-running campaigns (as you'll saturate their audience):
+
+- [TLDR](https://tldr.tech/)
+- [Pointer](https://www.pointer.io/)
+- [Bytes](https://bytes.dev/), [React newsletter](https://reactnewsletter.com/)
+- [Tech Lead Digest](https://techleaddigest.net/), [Programming Digest](https://programmingdigest.net/)
+- [Software Lead Weekly](https://softwareleadweekly.com/)
+- [Architecture Notes](https://architecturenotes.co/)
+- [Hackernewsletter](https://hackernewsletter.com/)
+
+Smaller newsletters that we also have supported:
+
+- [Level Up](https://levelup.patkua.com/)
+- [Console](https://console.dev/)
+- [FOSS Weekly](https://fossweekly.beehiiv.com/)
+
+### A couple we haven't tried yet
+
+Over the next few months, we're planning to try out paid ads on G2 and Stack Overflow. We think G2 will be useful for reaching larger companies who are running more formal processes to replace their tools by trying several options first. Stack Overflow we have heard is (anecdotally) good for general awareness-building, but not conversion. Both are quite expensive but very focused options to consider for dev tools. I'll update this article with our findings!
+
+## What next?
```

Comment:
```suggestion
## How to get started with paid ads
```
What's next seems like you are trying to get me to read something else or do a CTA, but this is better and important so a change of title might help!